### PR TITLE
Add wrapper for libvlc_media_player_set_xwindow

### PR DESCRIPTION
**Pull request by patknight**
Tell the player which X window to draw in. Allows embedding libvlc media into X apps (in my case a GTK+ app.)

Tested in my application - movie played correctly.

### CODE REVIEW

File: player.go [hunk 1]
```diff
@@ -299,3 +299,12 @@ func (p *Player) setMedia(m *Media) error {
 	C.libvlc_media_player_set_media(p.player, m.media)
 	return getError()
 }
+
+// SetXWindow sets the X window to play on
```
Comment (adrg) [inline player.go]: Can you please add a "." at the end of the comment sentence?
```
// SetXWindow sets the X window to play on.
```

File: player.go [hunk 2]
```diff
@@ -299,3 +299,12 @@ func (p *Player) setMedia(m *Media) error {
...
+func (p *Player) SetXWindow(XID uint32) error {
```
Comment (adrg) [inline player.go]: Please consider calling the parameter `windowID` instead of `XID`.

File: player.go [hunk 3]
```diff
@@ -299,3 +299,12 @@ func (p *Player) setMedia(m *Media) error {
...
+	if p.player == nil {
+		return errors.New("A player must be initialized first")
+	}
+	C.libvlc_media_player_set_xwindow(p.player, C.uint(XID))
```
Comment (adrg) [inline player.go]: Please add an empty line after the `if` statement for consistency with the code base.

File: player.go (hunk 1 above)
Comment (patknight) [reply player.go]: Done

File: player.go (hunk 2 above)
Comment (patknight) [reply player.go]: Done

File: player.go (hunk 3 above)
Comment (patknight) [reply player.go]: Done

**Comment by patknight** [comment]
Thanks for the feedback. All addressed. I though I'd maintained coding style, but missed those. Sorry.

**Comment by adrg** [comment]
Great. Thank you again for your PR. I'll add you to the contributors list in the README if you don't mind.

**Pull request merged**